gapic-generator-cloud/lib/gapic/presenters/wrapper_gem_presenter.rb

```diff
@@ -106,13 +106,15 @@ def default_versioned_gem
       end
 
       def dependencies
-        deps = { "google-cloud-core" => "~> 1.5" }
-        version_dependencies.each do |version, requirement|
-          deps["#{name}-#{version}"] = "~> #{requirement}"
+        @dependencies = begin
```

Comment:
Did you mean to memoize this? Generally we haven't done that in presenters, but if you think we need to do it here, you probably meant `@dependencies ||= begin`

Reply:
right, fixed